axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveTemplateService.java

```diff
@@ -267,6 +267,8 @@ public List<Long> generateMove(MoveTemplate moveTemplate, List<HashMap<String, O
   public List<Long> generateMove(LocalDate moveDate, List<HashMap<String, Object>> moveTemplateList)
       throws AxelorException {
     List<Long> moveList = new ArrayList<>();
+    Partner moveTemplatePartner = null;
```

Comment:
Please remove this line and declare the variable when it is retrieved from the method fillPartnerWithMoveTemplate(moveTemplate);

Reply:
Fixed